front-end/src/components/Results.js

```diff
@@ -125,13 +124,31 @@ const ResultsPage = () => {
                             between={2}
                             total={recordCount}
                             //leave as magic number idk why it works :/
-                            limit={10}
+                            limit={recordsPerPage}
                             last={true}
-                            changePage={(num) => handlePageChange(num)}
+                            changePage={(num) => setActivePage(num)}
                             ellipsis={1}
                         />
                     </Col>
                 </Row>
+                <Row>
+                    <Col className='pagination'>
+                        <DropdownButton id='perPageSelect' title={recordsPerPage}>
+                            <Dropdown.Item key='5' value='5' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>5</Dropdown.Item>
+                            <Dropdown.Item key='10' value='10' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>10</Dropdown.Item>
+                            <Dropdown.Item key='15' value='15' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>15</Dropdown.Item>
+                            <Dropdown.Item key='20' value='20' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>20</Dropdown.Item>
```

Comment:
should also update the active page, to 1 or do some math to get the current page based on the new per page